client/src/game/ui/selection/shapecontext.vue

```diff
@@ -50,6 +50,7 @@
         <li @click="addInitiative">{{ getInitiativeWord() }} initiative</li>
         <li @click="deleteSelection">Delete shapes</li>
         <li v-if="hasSingleShape()" @click="openEditDialog">Show properties</li>
+        <li v-if="hasSingleShape()" @click="setMarker">Set marker</li>
```

Comment:
After playing with it, we might want to hide this if it already is a marker or maybe replace it with a remove marker element.

Reply:
Right, I've been trying this. vue side is ok, gamestore side is ok. I can query the markers array in the gamestore from vue but I can't get the result back to vue, it's always undefined. Do I need to export something from gamestore or do I need to query the db directly from vue?

Reply:
You could add a method to the gamestore, but you do have everything you need in this file.  You import the markers from the store in your computed section and you have the access to the shape and thus it's id.